packages/docsearch-react/src/ResultsScreen.tsx

```diff
@@ -15,7 +16,9 @@ export function ResultsScreen(props: ResultsScreenProps) {
           return null;
         }
 
-        const title = collection.items[0].hierarchy.lvl0;
+        const title = unescape(
+          collection.items[0]._highlightResult.hierarchy_camel[0].lvl0.value
```

Comment:
Why do we use the camel version?

Reply:
I matched the v2 implementation, is there any differences/preferences expect that `camel` is an array?

Reply:
replaced in https://github.com/algolia/docsearch/pull/1001/commits/5ea358bd4b441c74ae83a1573ba9ab81370d7261 as we discussed with @Shipow